stdlib/list.mli

```diff
@@ -230,6 +230,15 @@ val find_opt: ('a -> bool) -> 'a list -> 'a option
     satisfies [p] in the list [l].
     @since 4.05 *)
 
+val find_some: ('a -> 'b option) -> 'a list -> 'b option
+(** [find_some f l] applies [f] to every element of [l] in turn,
```

Comment:
I prefer "each element" to "every element" here, since the traversal stops once `f` "succeeds".